contracts/contracts/strategies/CompoundStrategy.sol

```diff
@@ -14,6 +15,21 @@ import {
 contract CompoundStrategy is InitializableAbstractStrategy {
     event SkippedWithdrawal(address asset, uint256 amount);
 
+    /**
+     * @dev Collect accumulated COMP and send to Vault.
+     */
+    function collectRewardToken() external onlyVault nonReentrant {
+        // Claim COMP from Comptroller
+        ICERC20 cToken = _getCTokenFor(assetsMapped[0]);
+        IComptroller comptroller = IComptroller(cToken.comptroller());
+        comptroller.claimComp(address(this));
+        // Transfer COMP to Vault
+        IERC20 rewardToken = IERC20(rewardTokenAddress);
+        uint256 balance = rewardToken.balanceOf(address(this));
+        emit RewardTokenCollected(vaultAddress, balance);
```

Comment:
Should we have a conditional on balance > 0 so that we don't emit the event / do the transfer for zero amount?

Reply:
Separately, it seems it would be useful interpreting those events to include the address of the reward token in the event. It can be figured out by looking up strategy.rewardToken, but still.
Anyway, not a blocker for this PR. More of a possible tweak for future.

Reply:
#519